data-sources/vercel/config.yml

```diff
@@ -0,0 +1,31 @@
+id: vercel
+displayName: Vercel
+description: |
+  ## About Vercel
+
+  Vercel is the platform for frontend developers, providing the speed and reliability innovators need to create at the moment of inspiration.
+
+  We enable teams to iterate quickly and develop, preview, and ship delightful user experiences. Vercel has zero-configuration support for 35+ frontend frameworks and integrates with your headless content, commerce, or database of choice.
+
+icon: logo.svg
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstartId: 
```

Comment:
We'll just need to get the generated vercel quickstart id and add it here

Reply:
@aswanson-nr got it, I removed the data source so it should be good now. I'll add it in later.